db/db.go

```diff
@@ -77,9 +77,6 @@ type Transaction interface {
 	// Get fetches the value for the given key, should return ErrKeyNotFound if key is not present
 	// Caller should not assume that the slice would stay valid after the call to cb
 	Get(key []byte, cb func([]byte) error) error
-
-	// Impl returns the underlying transaction object
-	Impl() any
```

Comment:
@IronGauntlets do you remember why this method was introduced in the first place? how is this different from `db.(customType)` ?

Reply:
I wasn't too sure about this when Omer implemented it. If it is not used anywhere we can delete it.